napari/_qt/qt_json_builder/README.md

```diff
@@ -0,0 +1,100 @@
+# qt-jsonschema-form
```

Comment:
I'm curious how others feel, but I'd probably say we don't need to include their readme. I guess one question is if we're just vendoring their code so we have a copy (in which case maybe it should live inside a `_vendored` or similar folder, or if we're going to iterate and develop on this code ourselves (like the PyQt5 -> qtpy change and more).

I see more of the later long term, in which case we probably don't want their readme, but instead will want to generate our own docs that cover our use cases. Curious what others think though

Reply:
I agree with your assessment @sofroniewn, sounds like @goanpeca does too. Thanks all!